src/nextjournal/clerk/analyzer.clj

```diff
@@ -1,5 +1,4 @@
 (ns nextjournal.clerk.analyzer
-  {:nextjournal.clerk/no-cache true}
```

Comment:
Now that no-cache inheritance is fixed (#633) we need to remove this, otherwise expressions using a bunch of built-in viewers like `clerk/table` would not be cached.